examples/react-design-system/src/builder-settings.js

```diff
@@ -1,4 +1,5 @@
 import { Builder, builder } from '@builder.io/react';
+import '@builder.io/widgets';
```

Comment:
if we want this in we should add a comment here that adding builder.io/widgets adds a carousel, tabs, and slider component for use in the editor

Reply:
I ended up removing it

Reply:
great